src/suite_starter/suite_starter.py

```diff
@@ -86,9 +91,22 @@ def _configure(self):
             "ttl": os.getenv("ETOS_ESR_TTL", "3600"),
             "termination_grace_period": os.getenv("ETOS_TERMINATION_GRACE_PERIOD", "300"),
             "sidecar_image": os.getenv("ETOS_SIDECAR_IMAGE"),
+            "otel_collector_host": os.getenv("OTEL_COLLECTOR_HOST"),
```

Comment:
What happens if OTEL_COLLECTOR_HOST is not set? ETOS must work even if those variables are not set.

Reply:
Made adjustments so that it works without errors in the log.